o11y/profiler/datadog/datadog.go

```diff
@@ -15,13 +15,16 @@ type Sink struct{}
 
 // New initiated datadog profiler sink
 func New() (Sink, error) {
-	err := profiler.Start(profiler.WithProfileTypes(
-		profiler.CPUProfile,
-		profiler.HeapProfile,
-		profiler.BlockProfile,
-		profiler.MutexProfile,
-		profiler.GoroutineProfile,
-	))
+	err := profiler.Start(
+		profiler.WithProfileTypes(
+			profiler.CPUProfile,
+			profiler.HeapProfile,
+			profiler.BlockProfile,
+			profiler.MutexProfile,
+			profiler.GoroutineProfile,
+		),
+		profiler.WithLogStartup(false),
```

Comment:
Why disabling this?

Reply:
the startup log does not go through our usual logger (and I couldn't parameter this) and is picked up as an error -- which is confusing. Same with the tracer !